data/ngoApplyData.js

```diff
@@ -104,6 +104,17 @@ export default {
           happen around the middle of the semester and the handoff, which will
           be around the end of the semester. For more information, look{" "}
           <a href="https://h4i-project-timeline-qonwikgcqk.now.sh">here.</a>
+          <br/>
+          <br/>
+          In most cases, projects and development will span one semester, after which
+          our team will transfer ownership of the repository and any additional resources 
+          created during the development process. More recently, however, some projects have 
+          spanned the course of an entire year in an effort to ensure quality and impact. 
+          The development process follows the same structure in these cases, where a 
+          fully functional prototype is completed after one semester. However, these projects 
+          continue by gathering feedback during the next semester and iterating on the 
+          prototype to ensure that we're creating the most impactful and useful product for our 
+          clients.
```

Comment:
We should mention that this is usually a decision we come to as a group. Not like something they pick on their own.

Reply:
Also maybe just saying the two options as equal rather than saying `in most cases` for one, and recently for the other.

Reply:
replacing `in most cases` with `traditionally` would also be fine imo.